### DISCUSSION TITLE
Is there any way to get record by user_id instead of record id, just as we have in mongodb

### DISCUSSION THREAD

**Discussion by rajesh6161** [pocketbase/pocketbase#1002] [Q&A] [answered]

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure that I understand what do you refer to.

If `user_id` is another field in your collection, in the upcoming v0.8.0 you'll be able to get the first list item matching a filter expression using:
```js
pb.collection("YOUR_COLLECTION").getFirstListItem(`user_id="${YOUR_USER_ID}"`)
```

You can emulate the above in v0.7.x using:
```js
pb.records.getList("YOUR_COLLECTION", 1, 1, { filter: `user_id="${YOUR_USER_ID}"` })
```

**Comment by rajesh6161** [comment]
Thanks it worked like a charm
